hasjob/forms/campaign.py

```diff
@@ -24,7 +24,7 @@ class CampaignContentForm(forms.Form):
         validators=[forms.validators.Optional(), forms.validators.AllUrlsValid()])
     banner_image = forms.URLField(__("Banner image URL"), validators=[forms.validators.Optional()],  # TODO: Use ImgeeField
         description=__("An image to illustrate your campaign"))
-    banner_location = forms.RadioField(__("Banner location"), choices=BANNER_LOCATION.items(), coerce=int,
+    banner_location = forms.RadioField(__("Banner location"), choices=list(BANNER_LOCATION.items()), coerce=int,
```

Comment:
In this case, `list()` is necessary as we are replacing a frame object with a list.